### PR TITLE
client: added an example of snowball tracing with a txn

### PR DESCRIPTION
**Pull request by vivekmenezes**
I thought this was worth adding if anyone wanted to know how to do it outside of SQL.

---
This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/12002)

### CODE REVIEW

**Comment by RaduBerinde** [comment]
LGTM

File: pkg/internal/client/txn_test.go [hunk 1]
```diff
@@ -39,6 +43,39 @@ var (
...
+	log.Event(ctx, "txn complete")
+	// Cannot use ctx once Finish() is called.
+	sp.Finish()
+	dump := tracing.FormatRawSpans(collectedSpans, false /*formatTiming*/)
+	fmt.Printf("trace:\n%s", dump)
+	// Output:
+	// trace:
+	//     event:collecting spans
+	//     event:client.Txn did AutoCommit. err: <nil>
+	// txn: "internal/client/txn_test.go:63 Example_txnSnowballTrace" id=<nil> key=/Min rw=false pri=0.00000000 iso=SERIALIZABLE stat=COMMITTED epo=0 ts=0.000000000,0 orig=0.000000000,0 max=0.000000000,0 wto=false rop=false
```
Comment (vivekmenezes) [inline pkg/internal/client/txn_test.go]: I'm updating this to be a test in a separate PR
